src/chi/components/steps/steps.scss

```diff
@@ -87,5 +97,100 @@
         color: $brand-color;
       }
     }
+
+    &.-inverse {
+      background-color: set-color(black);
+      padding: 1rem;
+
+      & > li {
+        &::before {
+          background-color: $brand-color;
+          border: 0.125rem solid set-color(black);
+        }
+
+        &.-active {
+          & ~ li {
+            &::before {
+              background-color: set-color(grey, 40);
+              border-color: set-color(black);
+            }
+
+            & a {
+              color: $muted-color;
+            }
+          }
+        }
+
+        & a {
+          color: $brand-color;
+        }
+      }
+
+      &.-horizontalLabels {
+        counter-reset: step;
+
+        & > li {
+          &::before {
+            background-color: set-color(black);
+            border: 0.125rem solid $brand-color;
+            counter-increment: step;
+            content: counter(step);
+            height: 1.3rem;
+            margin: 0 auto;
+            width: 1.3rem;
```

Comment:
Add `font-weight: 600;` and change font-size of numbers to `font-size: 0.8125rem;`